tsl/src/continuous_aggs/materialize.c

```diff
@@ -298,31 +292,19 @@ get_materialization_end_point_for_table(int32 raw_hypertable_id, int32 materiali
 	Dimension *time_column = hyperspace_get_open_dimension(raw_table->space, 0);
 	NameData time_column_name = time_column->fd.column_name;
 	Oid time_column_type = ts_dimension_get_partition_type(time_column);
-	bool found_new_tuples = false;
+	int64 now_time = ts_get_now_internal(time_column);
```

Comment:
this function could be set to find `max` when there's an integer time-type and no integer `now` function, which would prevent this PR from breaking current users.

Reply:
discussed offline